.github/workflows/codeql.yml

```diff
@@ -0,0 +1,43 @@
+name: "CodeQL"
+
+on:
+  push:
+  pull_request:
```

Comment:
This results in the workflow running twice on pull requests.
Please use a branch filter on the push event or remove the pull_request event.